src/sentry/performance_issues/detectors/experiments/n_plus_one_api_calls_detector.py

```diff
@@ -169,6 +169,13 @@ def _maybe_store_problem(self) -> None:
         if not fingerprint:
             return
 
+        if self.stored_problems.get(fingerprint):
+            logging.info(
+                "Multiple occurrences detected for fingerprint",
+                extra={"detector": self.settings_key},
+            )
+            return
+
```

Comment:
in your testing did you see any cases with more than 2 occurrences? would it be useful to log a total count? ofc we can always figure that out from these logs anyway